charts/kserve-resources/templates/configmap.yaml

```diff
@@ -199,9 +198,15 @@ data:
            # NOTE: This configuration only applicable for serverless deployment with Istio configured as network layer.
            "ingressGateway" : "knative-serving/knative-ingress-gateway",
 
-           # ingressService specifies the hostname of the ingress service.
+           # knativeLocalGatewayService specifies the hostname of the Knative's local gateway service.
+           # The default KServe configurations are re-using the Istio local gateways for Knative. In this case, this
+           # knativeLocalGatewayService field can be left unset. When unset, the value of "localGatewayService" will be used.
```

Comment:
This is not backwards compatible if user has customized the existing `localGatewayService`

Reply:
oh I see you have
```go
if len(ingressConfig.KnativeLocalGatewayService) == 0 {
 			ingressConfig.KnativeLocalGatewayService = ingressConfig.LocalGatewayServiceName
 		}
```

Reply:
Yes, the code is what makes it backwards compatible.